flake8_adjustable_complexity/checker.py

```diff
@@ -68,3 +68,17 @@ def run(self) -> Generator[Tuple[int, int, str, type], None, None]:
                 self._error_message_template.format(actual_complexity, max_expected_complexity),
                 type(self),
             )
+
+    @classmethod
+    def add_options(cls, parser) -> None:
+        parser.add_option(
+            '--max-mccabe-complexity',
+            type=int,
+            parse_from_config=True,
+            help='Max mccabe complexity',
+            default=cls.DEFAULT_MAX_MCCABE_COMPLEXITY,
+        )
+
+    @classmethod
+    def parse_options(cls, options) -> None:
+        cls.DEFAULT_MAX_MCCABE_COMPLEXITY = int(options.max_mccabe_complexity)
```

Comment:
`cls.DEFAULT_MAX_MCCABE_COMPLEXITY` is wrong name now. It should be not upper-case and without "default" prefix (`cls.max_mccabe_complexity`).

Reply:
Yes. All right :)